src/main/java/com/exactpro/remotehand/windows/actions/GetScreenshot.java

```diff
@@ -16,41 +16,41 @@
 
 package com.exactpro.remotehand.windows.actions;
 
-import com.exactpro.remotehand.ActionOutputType;
 import com.exactpro.remotehand.ScriptExecuteException;
+import com.exactpro.remotehand.screenwriter.ScreenWriter;
+import com.exactpro.remotehand.screenwriter.SourceScreenWriter;
 import com.exactpro.remotehand.windows.ElementSearcher;
 import com.exactpro.remotehand.windows.WindowsAction;
 import com.exactpro.remotehand.windows.WindowsDriverWrapper;
 import com.exactpro.remotehand.windows.WindowsSessionContext;
+import io.appium.java_client.windows.WindowsDriver;
 import org.openqa.selenium.WebElement;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Base64;
 import java.util.Map;
 
 public class GetScreenshot extends WindowsAction {
 
-	private static final Logger loggerInstance = LoggerFactory.getLogger(GetScreenshot.class);
-	
-	public static final String NAME_PARAM = "name";
-	
+	private static final Logger logger = LoggerFactory.getLogger(GetScreenshot.class);
+
 	@Override
 	public String run(WindowsDriverWrapper driverWrapper, Map<String, String> params, WindowsSessionContext.CachedWebElements cachedElements) throws ScriptExecuteException {
-		String screenshotName = params.get(NAME_PARAM);
-		ElementSearcher es = new ElementSearcher(params, this.getDriver(driverWrapper), cachedElements);
+		WindowsDriver<?> driver = this.getDriver(driverWrapper);
+		ElementSearcher es = new ElementSearcher(params, driver, cachedElements);
 		WebElement element = es.searchElement();
-		return element == null
-				? this.takeScreenshot(screenshotName)
-				: screenWriter.takeAndSaveScreenshot(screenshotName, element);
-	}
+		if (element == null) {
+			throw new ScriptExecuteException("Getting screenshot of whole screen is not available, because encoded " + 
```

Comment:
Replace message with the following:
"Getting screenshot of the whole screen is not available, because encoded screenshot data can be very large"